### PR TITLE
cleanup and renaming

### PR DESCRIPTION
**Pull request by zyndiecate**

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -1,11 +1,11 @@
 package apiclient
 
 import (
-	userConfig "github.com/giantswarm/user-config"
+	"github.com/giantswarm/user-config"
```
Comment (ewoutp) [inline config.go]: +1

File: service_definition.go [hunk 2]
```diff
@@ -9,13 +9,14 @@ import (
 )
 
 const (
-	INDENTION        = "  "
```
Comment (ewoutp) [inline service_definition.go]: Technically this breaks the interface of this repo, but I'm ok with it

**Comment by ewoutp** [comment]
LGTM

File: service_definition.go (hunk 2 above)
Comment (zyndiecate) [inline service_definition.go]: I think in practice it does not. It was nowhere used. Lets keep private things private.

**Pull request merged**